openmole/plugins/org.openmole.plugin.task.netlogo/src/main/scala/org/openmole/plugin/task/netlogo/NetLogoTaskBuilder.scala

```diff
@@ -22,6 +22,7 @@ import org.openmole.core.context.Val
 
 trait NetLogoTaskBuilder[T] {
   def netLogoInputs: Lens[T, Vector[(Val[_], String)]]
+  def netLogoArrayInputs: Lens[T, Vector[(Val[_], String)]]
```

Comment:
I am not sure this is requiered. I think it is possible to achieve the same thing without having a special setting.